code/modules/mob/mob_movement.dm

```diff
@@ -187,6 +187,10 @@
 
 /client/proc/Move_object(direct)
 	if(mob && mob.control_object)
+		if(istype(mob.control_object,/obj/item/projectile/nikita))

```

Comment:
Move_object was previously only used for admins Possessing objects. I'd rather not code an entirely new proc and add new vars just for the nikita, now THAT would be snowflaky.